dask/dataframe/io/parquet/core.py

```diff
@@ -222,6 +222,13 @@ def read_parquet(
     if index and isinstance(index, str):
         index = [index]
 
+    if filters:
+        # Filters require us to gather statistics
```

Comment:
You can filter on partition columns without gathering statistics?

Reply:
Ah - Good point